File: FastScript/Demos/Main/Samples/JScript/case.js
//****************************/
//* FastScript v1.0          */
//* 'Case' operator demo     */
//****************************/

var i, j;


  i = 0;

  switch (i)
  {
    case 1: j = 1;
    case 2..10: { j = 2; j++; }
    default: { j = 10; }
  }

  ShowMessage(j);

    

File: FastScript/Demos/Main/Samples/BasicScript/case.vb
' FastScript v1.0          
' 'Case' operator demo    


  dim i = 0, j

  select case i
    case 1: j = 1
    case 2..10: j = 2: Inc(j)
    case else: j = 10
  end select
  ShowMessage(j)

    